itsy-bitsy-data-structures.js

```diff
@@ -1412,11 +1415,6 @@ class BinarySearchTree {
 
         // Otherwise just move on to the left node.
         current = current.left;
-
-      // If the number isn't less than or greater, then it must be the same and
```

Comment:
I think this is actually really useful for what the fall through code would be doing. It is easy to bypass this if you aren't new to programming, but as a novice this is extremely useful to understand this. Just my thoughts.